src/variables.scss

```diff
@@ -5,6 +5,8 @@ $neutral: var(--qr-color-neutral, hsl(0, 0%, 45%));
 $neutral-light: var(--qr-color-neutral-light, hsl(0, 0%, 96%));
 $neutral-dark: var(--qr-color-neutral-dark, hsl(0, 0%, 32%));
 
+$border-neutral: hsl(from $neutral-light h s 80%);
```

Comment:
Discussion: Do you think we should tweak the styling on the question text? I have the feeling that it's getting a little lost in the context now.

Reply:
For me it's fine, I am afraid it can look over-styled otherwise. Do you have something in mind?

Reply:
<img width="894" alt="Screenshot 2024-05-15 at 10 39 23" src="https://github.com/openHPI/quiz-recap/assets/22977799/0d24fcb3-7cff-4400-913d-ce264cbb37c8">

> Do you have something in mind?

Basically applying the same styling... but I have to admit I don't like it very much.

Reply:
As discussed, we reverted the styles on the answers during the quiz.